docs/content/docs/grype/_index.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: "Beta - Grype Integration"
+title: "Grype Integration"
 linkTitle: "Grype Vulnerability Scanner"
 weight: 1
 ---
 
-Anchore Engine 0.10 includes a limited-functionality beta release of an integration with [Grype](https://github.com/anchore/grype)
-for vulnerability scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Engine,
-but is provided in this release in a preview capacity so users can try it out.
+As of Anchore Engine 1.0.0, Anchore Engine is fully integrated with Grype by default for vulnerability scanning. The V2 vulnerability scanner, based on Grype, replaces the legacy vulnerability scanner in previous versions of Anchore Engine.
+You can keep the legacy vulnerability scanner when installing Anchore Engine 1.0.0, but you will have to explicitly configure 1.0.0 to use the legacy vulnerability scanner.
 
-***Note:*** This tech preview is not intended for use in production environments. It should be installed in sandbox environments,
-and is strictly provided here to give users an early, hands-on preview of the feature. It may not include all 
-functionality of the legacy scanner. Please report any issues found with it on the
-[anchore-engine Github repo](https://github.com/anchore/anchore-engine/issues).
+If you are upgrading to Anchore Engine 1.0.0 from an earlier version, you will retain your previous vulnerability scanner setting. You will need to follow the linked instructions for upgrading to the new vulnerability scanner. 
+
+***Note:*** The legacy vulnerability scanner will be removed in a future release.
 
 ### Installing
-As of 0.10.0, Anchore Engine can be configured to use either the legacy or grype vulnerability scanner. It is not possible to run
-both vulnerability scanners at the same time. This configuration is picked up at bootstrap, and cannot be changed on a running system.
+As of Anchore Engine 1.0.0, the V2 vulnerability scanner, based on Grype, is included with Anchore Engine by default. 
 
-The grype scanner is intended for use in sandbox or staging environments in the current release. Downgrading from the
-grype scanner back to the legacy scanner is not supported and will cause data issues.
+### Installing and keeping the legacy feed
```

Comment:
"the legacy feed" -> "the legacy feed and vulnerability scanner"

Reply:
Good catch!